src/pds/registrysweepers/utils/__init__.py

```diff
@@ -143,14 +146,11 @@ def query_registry_db(
         total_hits = data["hits"]["total"]["value"]
         log.debug(f"   paging query ({served_hits} to {min(served_hits + page_size, total_hits)} of {total_hits})")
 
-        last_info_log_at_percentage = 0
-        log.info("Query progress: 0%")
-
         for hit in data["hits"]["hits"]:
```

Comment:
there is a `tqdm` package which can help to track progress in a loop. I would prefer to use that rather than adding specific indicators which can make the code less readable regarding its primary focus.